Gulpfile.js

```diff
@@ -0,0 +1,471 @@
+		.pipe(gulp.dest( path.join(__dirname, 'bower_components') ));
+});
+
+gulp.task('clean', function() {
+	return del([BUILD + '**', BUILD_DOCS+ '**']);
+});
+
+gulp.task('copy', function() {
+	return gulp.src([SRC + '**/*.+(html|js|woff)', '!' + SRC +'**/example.html'])
+		.pipe(changed(BUILD))
+		.pipe(debug())
+		.pipe(gulp.dest(BUILD));
+});
+
+gulp.task('sass', function() {
+	var wrapper = fs.readFileSync(TEMPLATES + "style_module_template.html").toString('utf8');
+	return gulp.src(SRC + 'mm-*/*.scss')
+		// .pipe(cache())
+		.pipe(changed(BUILD, {extension:'.css'}))
+		.pipe(sass({includePaths: ['./bower_components/bourbon/app/assets/stylesheets/', './src/shared/sass/']}).on('error', sass.logError))
+		.pipe(postcss([autoprefixer({browsers: ['last 2 versions']})]))
+		.pipe(gulp.dest(BUILD))
+		// .pipe(wrap({src:TEMPLATES + "style_module_template.html"},{},{engine:"hogan"}))
+		.pipe(wrap(function(data) {
+			data.fname = path.basename(data.file.relative,'.css');
+			return wrapper;
+		},{},{engine:"hogan"}))
+		.pipe(rename({basename:"style", extname: ".html"}))
+		.pipe(gulp.dest(BUILD));
+});
+
+gulp.task('font', function() {
+	return gulp.src(SRC + 'shared/fonts/fonts.scss')
+		.pipe(sass({includePaths: ['./bower_components/bourbon/app/assets/stylesheets/', './src/shared/sass/']}).on('error', sass.logError))
+		.pipe(gulp.dest(BUILD + 'shared/fonts/'))
+		.pipe(wrap("<style>{{{contents}}}</style>",{},{engine:"hogan"}).on('error',console.log))
+		.pipe(rename("fonts.html").on('error',console.log))
+		.pipe(gulp.dest(BUILD + '/shared/fonts/'));
+});
+
+gulp.task('vulcanize', function() {
+	var modules = gulp.src(BUILD + "mm-*/mm-*.html")
+		.pipe(changed(BUILD))
+		.pipe(vulcanize({
+			inlineScripts: true,
+			inlineCss: true,
+			excludes: ['bower_components/polymer/polymer.html']
+		}))
+		.pipe(debug())
+		.pipe(htmlmin())
+		.pipe(gulp.dest(BUILD));
+
+	var lib = gulp.src(BUILD + "strand.html")
+		.pipe(vulcanize({
+			inlineScripts:true,
+			inlineCss:true,
+			stripExcludes:false,
+			excludes: ['bower_components/polymer/polymer.html']
+		}))
+		.pipe(gulp.dest(BUILD));
+	return merge(modules, lib);
+});
+
+gulp.task('default', function(cb) {
+	run('clean','copy',['sass','font'],'vulcanize',cb);
+});
+
+gulp.task('build', function(cb) {
+	run('copy',['sass','font'],'vulcanize',cb);
+});
+
+gulp.task('build:prod', ['patch-lib', 'build'], function() {
+	var modules = gulp.src(BUILD + 'mm-*/mm-*.html')
+		.pipe(vulcanize({
+			inlineScripts:true,
+			inlineCss:true,
+			stripExcludes:false
+		}))
+		.pipe(base64(['.woff']))
+		.pipe(htmlmin({
+			quotes: true,
+			empty: true,
+			spare: true
+		}))
+		.pipe(inlinemin())
+		.pipe(header('<!--\n' + fs.readFileSync('BANNER.txt').toString('utf8') + ' -->'))
+		.pipe(changed(DIST))
+		.pipe(gulp.dest(DIST));
+
+	var lib = gulp.src(BUILD + 'strand.html')
+		.pipe(vulcanize({
+			inlineScripts:true,
```

Comment:
so the module excludes should actually exclude the mm-components as well, we really only want to inline the styles + js into the markup, but leave all the dependencies as external.